.gitignore

```diff
@@ -12,3 +12,4 @@ version.json
 buildhub-lambda-*.zip
 .docker-build
 .metadata-*.json
+.records-kinto-build-hub-releases.json
```

Comment:
I'm surprised this wasn't already in the `.gitignore`. It's technically out-of-scope for this PR but I snuck it in anyway. YOLO!